### DISCUSSION TITLE
CH21 and NFT not received

### DISCUSSION THREAD

**Discussion by rizampi** [Chia-Network/chia-blockchain#18347] [Support]
Hi, on June 27, 2024 I sent 1 CH21 to the address you provided (https://www.chia.net/2022/12/08/the-holiday-nfts-are-here/), to receive 1 NFT . To date I have not received anything... Obviously the NFT filter is also set to hidden.
<img width="964" alt="Greenshot 2024-07-23 18 10 49" src="https://github.com/user-attachments/assets/92943ae3-39c9-4d17-a2c5-674f7e500734">

**Comment by BrandtH22** [comment]
Hey @rizampi, Unfortunately the opportunity to redeem your CH21 token for an NFT has passed. I understand that this is frustrating, but there’s no way for the team to recover the tokens sent to the redemption address since the window has closed.

These were our notices of its closure:
discord: https://discord.com/channels/1034523881404370984/1074726388650737694/1161358101765705800

Twitter:
https://x.com/jde5011/status/1711800979874873794
https://x.com/jde5011/status/1732155570763817378

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
